### ISSUE TITLE
DApp should indicate if a name is in the dictionary or not

### ISSUE THREAD

**Issue by Arachnid** [ethereum/ens-registrar-dapp#90]
It's reasonable to expect peoples' bidding behaviour to vary depending on how easy it is to guess that name is up for auction. It'd be nice if the DApp showed something to indicate if the current name is in the preimage dictionary.

**Comment by jadster** [comment]
I agree! I certainly saw significantly different outcomes on auctions I started, depending on whether the name was in English and simple enough to be in the pre-image dictionary or not.

I assume there are multiple different pre-image directories that the average user is exposed to, right? e.g. on etherscan...
